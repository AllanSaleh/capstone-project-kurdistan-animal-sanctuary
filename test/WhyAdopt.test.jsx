import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import WhyAdopto from "../components/WhyAdopt";

test("proparly renders the WhyAdopto component", () => {
  const tree = renderer.create(<WhyAdopto />).toJSON();
  expect(tree).toMatchSnapshot();
});
