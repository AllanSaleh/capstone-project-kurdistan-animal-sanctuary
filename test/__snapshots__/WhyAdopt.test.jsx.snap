// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the WhyAdopto component 1`] = `
<div
  className="bg-secondary py-6 "
>
  <h1
    className="text-center text-primary text-4xl font-bold font-Quicksand "
  >
    Why Adopt?
  </h1>
  <p
    className="text-center p-6  text-lg text-primary md:px-64 font-Quicksand "
  >
    Here we will write a bit about what we do and why and the benfit of adopting animals and helping them, Here we will write a bit about what we do and why and the benfit of adopting animals and helping them
  </p>
</div>
`;
